### PR TITLE
Project Loading Performance Improvements

### PR DESCRIPTION
**Pull request by RoyalSix**

### CODE REVIEW

File: src/Api.js [hunk 1]
```diff
@@ -182,7 +181,7 @@ export default class Api extends ToolApi {
   }
 
   writeCheckData(payload = {}, checkPath) {
-    const modifiedTimestamp = generateTimestamp();
+    const modifiedTimestamp = moment().toJSON();
```
Comment (mannycolon) [inline src/Api.js]: Why are we switching to moment? is moment fatster than the previous generateTimestamp? if so I would suggest using date-fns. Its better optimized and a smaller size library. Highly recommended by the Javascript community. 

For more details see: https://medium.com/@k2u4yt/momentjs-vs-date-fns-6bddc7bfa21e

https://www.npmjs.com/package/date-fns

**Comment by RoyalSix** [comment]
I am actually going to close this PR. It looks like my benchmark tests were invalid from before.
https://jsbench.me/1kk19g8gkp

File: src/Api.js (hunk 1 above)
Comment (RoyalSix) [reply src/Api.js]: Ok, I see. It also looks like `date-fns` does not have a toJSON method

**Comment by RoyalSix** [comment]
Actually there is still an optimization that can be done here. Updating.

**Pull request merged**